localenv/mock-account-servicing-entity/app/lib/requesters.ts

```diff
@@ -116,6 +118,52 @@ export async function createPeer(
     })
 }
 
+export async function createAutoPeer(
+  peerUrl: string,
+  assetId: string
+): Promise<CreateOrUpdatePeerByUrlMutationResponse | undefined> {
+  const createAutoPeerMutation = gql`
+    mutation CreateOrUpdatePeerByUrl($input: CreateOrUpdatePeerByUrlInput!) {
+      createOrUpdatePeerByUrl(input: $input) {
+        code
+        success
+        message
+        peer {
+          id
+          name
+          asset {
+            id
+            scale
+            code
+            withdrawalThreshold
+          }
+        }
+      }
+    }
+  `
+
+  const addedLiquidity = '10000' as unknown as bigint
+  const createPeerInput: { input: CreateOrUpdatePeerByUrlInput } = {
+    input: {
+      peerUrl,
+      assetId,
+      addedLiquidity
```

Comment:
Isn't that missing the http token?

Reply:
no, there is not http token in this request.
I believe this is taken care of in rafiki

Reply:
Indeed: https://github.com/interledger/rafiki/blob/111a5040d158fe345172ca875979ae978b9bf7d3/packages/backend/src/payment-method/ilp/auto-peering/service.ts#L96